src/GameStore.js

```diff
@@ -36,32 +36,40 @@ export class GameStore {
         return this.heroes
     }
     toggleShowItems = () => {
-        this.showItems = !this.showItems;
-        this.showSkills = false
-        this.showActions = false
-        this.activeAction = null
-        this.activeSkill = null
-        this.activeItem = null
+        runInAction(() => {
+            this.showItems = !this.showItems;
+            this.showSkills = false
+            this.showActions = false
+            this.activeAction = null
+            this.activeSkill = null
+            this.activeItem = null
+        })
     }
     toggleShowActions = () => {
-        this.showActions = !this.showActions;
-        this.showSkills = false
-        this.showItems = false
-        this.activeAction = null
-        this.activeSkill = null
-        this.activeItem = null
+        runInAction(() => {
+            this.showActions = !this.showActions;
+            this.showSkills = false
+            this.showItems = false
+            this.activeAction = null
+            this.activeSkill = null
+            this.activeItem = null
+        })
     }
     toggleShowSkills = () => {
-        this.showSkills = !this.showSkills
-        this.showActions = false;
-        this.showItems = false
-        this.activeAction = null
-        this.activeSkill = null
-        this.activeItem = null
+        runInAction(() => {
+            this.showSkills = !this.showSkills
```

Comment:
I guess this first line in all these blocks is the difference, so this could be outside of the reset function

Reply:
yes this was just to encapsulate state mutations in a `runInAction` to appease `mobx` warnings due to strict mode